_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and machine learning (ML) tools for OpenSearch. It provides data analysts with the following abilities:
```

Comment:
Not sure if we should be precise here: machine learning (Natural Language processing support for now). @ylwu-amzn  what do you think?

Reply:
Yes, agree, I think we should clarify that